packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -1069,10 +1069,16 @@ - (NSNumber *)totalDiskCapacityWithError:(out NSError * __autoreleasing *)error
 // '<ARCType> *__autoreleasing*' problem solution: https://stackoverflow.com/a/8862061/4337317
 - (NSNumber *)freeDiskStorageWithError:(out NSError * __autoreleasing *)error
 {
-  NSDictionary *results = [self documentFileResourcesForKeys:@[NSURLVolumeAvailableCapacityForImportantUsageKey]
+#if TARGET_OS_TV
+  NSURLResourceKey key = NSURLVolumeAvailableCapacityKey;
+#else
+  NSURLResourceKey key = NSURLVolumeAvailableCapacityForImportantUsageKey;
+#endif
+
+  NSDictionary *results = [self documentFileResourcesForKeys:@[key]
                                                        error:error];
 
-  return results[NSURLVolumeAvailableCapacityForImportantUsageKey];
+  return results[key];
```

Comment:
This will have to be backported to Swift if my PR https://github.com/expo/expo/pull/23943 lands first (not likely)

Reply:
👍